service/src/main/resources/application.yml

```diff
@@ -152,6 +152,14 @@ workspace:
       id: wm-default-spend-profile
       # The billing account workspace-dev has access to.
       billing-account-id: 01A82E-CA8A14-367457
+      cloud-platform: GCP
+    - id: facade00-0000-4000-a000-000000000000
```

Comment:
🤔 I want to make sure I'm grokking this -- is the idea to config a full billing/spend profile model here for every profile we want to have limits? So, BPM will have a full copy of the BP and WSM will as well? If that's the case, I worry that we'll end up in situations where WSM and BPM disagree about both the data in a billing profile and it's state (i.e, deleted, not deleted ,etc.).

I wonder if this could be made more targeted. Instead of config'ing out the entire BP, we could create a map of BP IDs to limit configs and layer those in just-in-time when the spend profile service returns its results.

Reply:
This is what I'm thinking as well-- I would rather pull the information from BPM the first time it is read and then store that in the map. This avoids duplication and also reduces the amount of code changes considerably.

Reply:
I think that's a good idea - I was mostly trying to conform to what the pre-existing methods and classes expected, which was requiring the full information.  I'll do another pass-through.

Reply:
So to understand… with @aherbst-broad 's suggestion we still duplicate the limits in both repos, but we don't duplicate the MRG information, and we can tell if the billing profile has actually been deleted or the user no longer has permissions on it, which seems like an excellent idea.

I do still wonder if there is a way to store the limits from BPM by storing thing them during workspace creation and thus avoid duplicating the limits also, but it seems like folks don't want want to figure out how to store them in a WSM database?

Reply:
Yeah I think we didn't want to deal with storing in a database.  But even if I cut it down to just the ID and take out all the MRG info, I don't see how that will let us tell if it's been deleted or not.

Reply:
I believe it's because when we are authorizing against a spend profile, we will still call BPM get the actual profile (which does permission checks) instead of just using a hardcoded copy of information from the config. But the limits themselves will always be available because they are in config.

Reply:
[This](https://github.com/DataBiosphere/terra-workspace-manager/blob/0ce9424ef54582128ef9b73fa58341c9f748ec7a/service/src/main/java/bio/terra/workspace/service/spendprofile/SpendProfileService.java#L60) is the part that's tripping me up - when we read in from the config, we convert the `SpendProfileModel` to a `SpendProfile` which requires all the extra azure coordinate information.  I tried to minimize changes to this since presumably in other parts of the code we need a real `SpendProfile`, but it seems like if I need a dummy `SpendProfileModel` that doesn't contain all the information it'll need a different config, or other awkward changes to accommodate both the "real" and "limits" spendprofile, or instead of initializing from the config simply read from it as needed.